src/test/java/com/puppycrawl/tools/checkstyle/checks/modifier/ModifierOrderCheckTest.java

```diff
@@ -68,7 +68,7 @@ public void testIt() throws Exception {
             "49:35: " + getCheckMessage(MSG_ANNOTATION_ORDER, "@MyAnnotation4"),
             "157:14: " + getCheckMessage(MSG_MODIFIER_ORDER, "default"),
         };
-        verify(checkConfig, getPath("InputModifier.java"), expected);
+        verify(checkConfig, getPath("InputModifierIt.java"), expected);
```

Comment:
I will change it accordingly but for very small files the change might result in deletion and creation when I commit and not a rename. Is that ok?